tests/tests.cabal

```diff
@@ -13,22 +13,16 @@ author:             The Liquid Haskell Developers
 flag stack
   default:     False
   description: Enable when building with stack to prevent tests
-               from using cabal-install.
-
-flag build-negs
-  default:     False
-  description: Whether to build negative tests. Useful with stack
-               which insists on building everything which is
-               buildable on first run.
+               from using cabal-install, and to mark all executables as
+               " buildable: False " by default
 
 executable test-driver
     if flag(stack)
       main-is:        Driver_stack.hs
     else
       main-is:        Driver_cabal.hs
 
-    -- TODO(matt) freeze versions
```

Comment:
Not freezing here means they pick up whatever is used in the rest of the project.